ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -1,12 +1,17 @@
 (ns cmr.ingest.services.bulk-update-service
   (:require
+   [camel-snake-kebab.core :as csk]
    [cheshire.core :as json]
    [clojure.java.io :as io]
+   [clojure.string :as str]
```

Comment:
Ok I changed it, but I think now if we're in any file that uses it, we should be consciously updating it.